lang/lib/FontGen.py

```diff
@@ -34,45 +36,97 @@ def __init__(self, utf, charListIndex, alternate = None):
+    # CustomCharacter('Ê', 54, 'E'),
     CustomCharacter('ě', 47, 'e'),
+    # CustomCharacter('Ě', 55, 'E'),
+    CustomCharacter('ë', 48, 'e'),
+    # CustomCharacter('Ë', 56, 'E'),
     CustomCharacter('ę', 49, 'e'),
-    CustomCharacter('É', 52, 'E'),
+    CustomCharacter('Ę', 57, 'E'),
     CustomCharacter('í', 60, 'i'),
+    CustomCharacter('Í', 64, 'I'),
+    # CustomCharacter('ì', 61, 'i'),
+    # CustomCharacter('Ì', 65, 'I'),
     CustomCharacter('î', 62, 'i'),
     CustomCharacter('Î', 66, 'I'),
     CustomCharacter('ĺ', 68, 'l'),
+    # CustomCharacter('Ĺ', 72, 'L'),
     CustomCharacter('ł', 69, 'l'),
-    CustomCharacter('ľ', 70, 'l'),
     CustomCharacter('Ł', 73, 'L'),
+    CustomCharacter('ľ', 70, 'l'),
     CustomCharacter('Ľ', 74, 'L'),
     CustomCharacter('ń', 76, 'n'),
+    # CustomCharacter('Ń', 80, 'N'),
     CustomCharacter('ň', 77, 'n'),
-    CustomCharacter('ñ', 78, 'n'),
+    # CustomCharacter('Ň', 81, 'N'),
+    # CustomCharacter('ñ', 78, 'n'),
+    # CustomCharacter('Ñ', 82, 'N'),
     CustomCharacter('ó', 84, 'o'),
+    CustomCharacter('Ó', 92, 'O'),
     CustomCharacter('ò', 85, 'o'),
+    # CustomCharacter('Ò', 93, 'O'),
     CustomCharacter('ô', 86, 'o'),
+    # CustomCharacter('Ô', 94, 'O'),
+    # CustomCharacter('œ', 87, 'o'),
+    # CustomCharacter('Œ', 95, 'O'),
     CustomCharacter('ø', 88, 'o'),
+    CustomCharacter('Ø', 96, 'o'),
+    CustomCharacter('Ö', 97, '\\xef'),
     CustomCharacter('ő', 90, 'o'),
+    # CustomCharacter('Ő', 98, 'O'),
+    # CustomCharacter('ŕ', 100, 'r'),
+    # CustomCharacter('Ŕ', 104, 'R'),
     CustomCharacter('ř', 101, 'r'),
+    CustomCharacter('Ř', 105, 'R'),
     CustomCharacter('ś', 108, 's'),
+    CustomCharacter('Ś', 112, 'S'),
     CustomCharacter('š', 109, 's'),
+    CustomCharacter('Š', 113, 'S'),
     CustomCharacter('ș', 110, 's'),
+    CustomCharacter('Ș', 114, 'S'),
     CustomCharacter('ß', 111, '\\xe2'),
+    # CustomCharacter('ẞ', 115, '\\xe2'),
     CustomCharacter('ť', 116, 't'),
+    # CustomCharacter('Ť', 118, 'T'),
     CustomCharacter('ț', 117, 't'),
-    CustomCharacter('Ț', 119, 'T'),
+    # CustomCharacter('Ț', 119, 'T'),
     CustomCharacter('ú', 120, 'u'),
+    # CustomCharacter('Ú', 128, 'U'),
+    # CustomCharacter('ù', 121, 'u'),
+    # CustomCharacter('Ù', 129, 'U'),
+    # CustomCharacter('û', 122, 'u'),
+    # CustomCharacter('Û', 130, 'U'),
+    CustomCharacter('Ü', 133, 'U'),
     CustomCharacter('ů', 123, 'u'),
+    # CustomCharacter('Ů', 131, 'U'),
     CustomCharacter('ű', 124, 'u'),
+    # CustomCharacter('Ű', 132, 'U'),
     CustomCharacter('ý', 136, 'y'),
+    CustomCharacter('Ý', 138, 'Y'),
+    # CustomCharacter('ÿ', 137, 'y'),
+    # CustomCharacter('Ÿ', 139, 'Y'),
     CustomCharacter('ź', 140, 'z'),
+    # CustomCharacter('Ź', 144, 'Z'),
     CustomCharacter('ž', 141, 'z'),
+    CustomCharacter('Ž', 145, 'Z'),
     CustomCharacter('ż', 142, 'z'),
+    # CustomCharacter('Ż', 146, 'Z'),
+    # CustomCharacter('¿', 254, ' '),
```

Comment:
Shouldn't the  `¿ ¡` be replaced with anything?

Reply:
I added these here since I had them in my font. We don't use them since these unicode characters get preprocessed to an empty string, so effectively removed. I have them here just for reference. We can't use an empty character sadly as the alternate form, so this is the best I could make with the available codebase. In any case, it's disabled, so not something to worry about.

Reply:
Can confirm that the Spanish wizard only has one Space in 2nd part of the message which is a question.